src/main/java/com/laytonsmith/core/events/Event.java

```diff
@@ -59,7 +59,7 @@
 	 * Error.
 	 */
 	@Deprecated
-	public boolean matches(Map<String, Mixed> prefilter, BindableEvent e) throws PrefilterNonMatchException;
+	public boolean matches(Map<String, Mixed> prefilter, BindableEvent e, Environment env) throws PrefilterNonMatchException;
```

Comment:
matches(), evaluate(), modifyEvent() (and maybe event convert()?) changes all break extensions.

Reply:
I'm skipping convert, but everything else is using reflection to recall the method now.